pkg/cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -48,9 +48,14 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 			return deleteCluster(logger, flags)
 		},
 	}
+
+	var (
+		name = "name"
```

Comment:
Nit: This should be a `const` not a var and it should have a more descriptive name or else just explicitly paste `"name"` into each location. It's not really clearer to have `const name = "name" and it's strictly worse to have it as a var.

Reply:
Thanks. Updated.